### PR TITLE
[ WIP ] Dx test

### PR DESCRIPTION
**Pull request by pierr**
- [x] Add a logger on each dispatcher
- [x] Add a DX store content display component
- [x] Add a store container in focus-core (as for now it is a custom wrapper in the demo)
- [ ] Add a switch between the two modes
- [ ] Add this into the layout and have a **DEV** mode
### Dispatch logger

image
### Store DX

image

### CODE REVIEW

**Comment by pierr** [comment]
@Bernardstanislas @Tommass Any opinion?

**Comment by pierr** [comment]
Ugly hack on focus core 

``` js
window._stores =  [];

var CoreStore = function (_EventEmitter) {
    _inherits(CoreStore, _EventEmitter);

    /**
    * Contructor of the store class.
    */

    function CoreStore(config) {
        _classCallCheck(this, CoreStore);

        var _this = _possibleConstructorReturn(this, Object.getPrototypeOf(CoreStore).call(this));

        assign(_this, {
            config: config
        });
        //Initialize the data as immutable map.
        _this.data = Immutable.Map({});
        _this.status = Immutable.Map({});
        _this.error = Immutable.Map({});
        _this.pendingEvents = [];
        _this.customHandler = assign({}, config.customHandler);
        //Register all gernerated methods.
        _this.buildDefinition();
        _this.buildEachNodeChangeEventListener();
        _this.registerDispatcher();
        window._stores.push(_this);
        return _this;
    }}
```

**Comment by pierr** [comment]
image
Routes

File: app/containers/dock-dx.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React, { cloneElement, Children, Component, PropTypes } from 'react';
+import Dock from 'react-dock';
+import parseKey from 'parse-key';
+import JSONTree from 'react-json-tree'
+
+export default class DockDX extends Component {
```
Comment (TomGallon) [inline app/containers/dock-dx.js:6]: Should we prefix all dx component as DxComponent ? 
in this case DxDock ?

Comment (pierr) [inline app/containers/dock-dx.js:6]: I think When they will be included in focus-components/Dev-tools  or in a focus-devtools repository then in the import I am ok with you

**Comment by pierr** [comment]
Done in #113

**Pull request closed without merging**